unix/vncserver/selinux/vncsession.te

```diff
@@ -72,13 +72,13 @@ optional_policy(`
 	userdom_spec_domtrans_all_users(vnc_session_t)
 	userdom_signal_all_users(vnc_session_t)
 
-	userdom_user_home_dir_filetrans(vnc_session_t, vnc_home_t, dir, ".vnc")
-	userdom_admin_home_dir_filetrans(vnc_session_t, vnc_home_t, dir, ".vnc")
+	userdom_user_home_dir_filetrans(vnc_session_t, vnc_home_t, dir)
+	userdom_admin_home_dir_filetrans(vnc_session_t, vnc_home_t, dir)
```

Comment:
Hmm... I'm not sure these are correct anymore. `.config` and friends are not tagged as `user_home_t`.

@grulja, @zpytela, do you have some insight here? This might also mean #1425 needs to be adjusted.

Reply:
Hmm... Thinking a bit more on this, do we even need the complexity of a custom type now that we use the XDG directories? Can't we just use the standard `config_home_t`? @zpytela?

Reply:
It makes sense to have private types and keep them separated if some domain needs to manage its data/config files or if there are some sensitive data. For reading it usually is not worth bothering.

Reply:
`vncsession` will only read out of `~/.config`, so no need for it there then. It will write to `.local/state/tigervnc`, but only the log file.

Looking at gdm, it wants to create `.cache/gdm/session.log`, and I can see that `.cache/gdm` is tagged as `xdm_home_t`.

That directory is specific to gdm, though, not a general GNOME directory. So I'm not sure that it's equivalent.

Looking at my home directory, only PulseAudio and libvirt have custom types for `.config`. And only Telepathy and GNOME keyring for `.local/share`. So it doesn't seem to be particularly common for custom types?

My `~/.local/state`  is also `gconf_home_t`, something it seems to have inherited from `~/.local`. Is that really correct? Or some historical artefact we'll have to live with? :/

@zpytela?

Reply:
One advantage of not having our own type is that the different XDG directories retain their type separation. Otherwise, our config, cache, data, and state will all have the same type.